### PR TITLE
framework/iotbus: Add Kconfig option for iotbus SPI/I2C

### PR DESCRIPTION
**Pull request by an4967**
- CONFIG_[SPI/I2C]_USERIO should be needed to run iotbus SPI/I2C on user space.
- update defconfig of TCs.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1ab073daf11c221fdc953ab8d2394aac3e99fbc5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1ab073daf11c221fdc953ab8d2394aac3e99fbc5] - Code Rule Check (C++) OK.

File: build/configs/esp32_DevKitC/tc/defconfig [hunk 1]
```diff
@@ -348,13 +348,15 @@ CONFIG_DEV_NULL=y
 # CONFIG_PWM is not set
 # CONFIG_ARCH_HAVE_I2CRESET is not set
 CONFIG_I2C=y
+CONFIG_I2C_USERIO=y
```
Comment (sangwon03) [inline build/configs/esp32_DevKitC/tc/defconfig]: This already have CONFIG_I2C_USERIO in 353L

File: build/configs/cy4390x/tc/defconfig [hunk 2]
```diff
@@ -352,13 +352,15 @@ CONFIG_DEV_ZERO=y
 CONFIG_PWM=y
 # CONFIG_ARCH_HAVE_I2CRESET is not set
 CONFIG_I2C=y
+CONFIG_I2C_USERIO=y
```
Comment (sangwon03) [inline build/configs/cy4390x/tc/defconfig]: This already have CONFIG_I2C_USERIO in 357L

**Pull request closed without merging**